hanko-js/src/lib/HankoClient.ts

```diff
@@ -295,6 +297,8 @@ class WebauthnClient extends AbstractClient {
           throw new WebAuthnRequestCancelledError(e);
         })
         .then((attestation: PublicKeyCredentialWithAttestationJSON) => {
+          // @ts-ignore
+          attestation.transports = attestation.response.transports;
```

Comment:
@FreddyDevelop do you have any clue what we can do here?

Reply:
Well, you can leave it like it is and add a comment why this is needed.  
You can also create a new type and use it to send the attestation.

Both ways are acceptable for me.

Reply:
Hmm, what should the comment say and what is reason the types do not match? Or does it behave as expected?

Reply:
Alright, thanks a lot! I did both, added a new interface and a comment.